### PR TITLE
#9 layout

### PR DESCRIPTION
**Pull request by Marc-elus**
Still working on this code

### CODE REVIEW

File: app/resources/styles.tsx [hunk 1]
```diff
@@ -46,4 +52,21 @@ export const DropdownContainer = styled.div`
...
     margin: 0;
-    `
\ No newline at end of file
+    `
+    export const VideoCard  = styled.div <Props>`
+    width: 250px;
+    height: 250px;
+    background-color: #D1D1D1;
+    border-radius: 20px;
+    background-image: url(${props => props.image || ''});
```
Comment (PilaPina) [inline app/resources/styles.tsx:61]: Is the empty '' about the image issues you had?

**Review by PilaPina — CHANGES_REQUESTED** [review]
Looks good. Great work. I had a question about the missing image. DId you want to add it before merge or just next time?

File: app/resources/styles.tsx (hunk 1 above)
Comment (Marc-elus) [reply app/resources/styles.tsx:61]: The image link its supposed to be in the empty commas

**Review by PilaPina — APPROVED** [review]
Great work.  
Can't wait to merge this. Like the video card component.

**Pull request merged**